app-config.yaml

```diff
@@ -15,7 +15,7 @@ proxy:
     target: 'https://circleci.com/api/v1.1'
     changeOrigin: true
     pathRewrite:
-      '^/circleci/api/': '/'
+      '^/proxy/circleci/api/': '/'
```

Comment:
I wonder - the `/api` part here feels like an old remnant from the package.json driven proxying, it should probably be removed

Reply:
Maybe, maybe not :) kinda shows ultra explicitly that we are proxying to circleci api